app/views/job_templates/auto_complete_job_category.json.erb

```diff
@@ -0,0 +1,3 @@
+[
+  <%= @job_categories.map { |name| '{"completed": "","part": "' + name + '","label": "' + name + '","category": ""}' }.join(',').html_safe %>
```

Comment:
Shouldn't we use a hash serialized to json via some JSON library: it's not that hard to create a name that would break this code as we're missing escaping of the values.